js/index.js

```diff
@@ -1 +1,76 @@
 // Your code goes here
+
+// listener type 1: mouseover
+// listerer type 2: mouseout
+// "Nest two similar events somewhere in the site and prevent the event propagation properly"
+const navElement = document.querySelector("nav");
+navElement.addEventListener("mouseover", event => {
+  navElement.style.backgroundColor = "darkgrey";
```

Comment:
I know you mentioned this in standup last night but remember going forward that you need to apply your desired effects on the `event` object being passed in using `event.target` when relevant to make sure you only affect the acted-upon element